python/lsst/dax/apdb/cassandra/apdbMetadataCassandra.py

```diff
@@ -47,19 +47,10 @@ def __init__(self, session: Any, table_name: str, keyspace: str, read_profile: s
         self._write_profile = write_profile
         self._part = 0  # Partition for all rows
         self._preparer = PreparedStatementCache(session)
-        # _table_clause will be None when metadata table is not configured
-        self._table_clause: str | None = None
-
-        query = "SELECT count(*) FROM system_schema.tables WHERE keyspace_name = %s and table_name = %s"
-        result = self._session.execute(query, (keyspace, table_name), execution_profile=read_profile)
-        exists = bool(result.one()[0])
-        if exists:
-            self._table_clause = f"{quote_id(keyspace)}.{quote_id(table_name)}"
+        self._table_clause = f"{quote_id(keyspace)}.{quote_id(table_name)}"
```

Comment:
Would it be helpful to have a check here, to test whether the supplied keyspace and table name exist?

Reply:
I think it's better to skip this check, it is an additional round-trip to database and the likelihood that it is missing is very low. But if it is indeed missing, then things crash a moment later with a clear diagnostics.